src/fiatconnect/ReviewScreen.tsx

```diff
@@ -339,15 +368,28 @@ function getDisplayAmounts({
     cUSD: string | null
     cEUR: string | null
   }
+  feeEstimate?: FeeEstimateState
 }) {
   const cryptoType = normalizedQuote.getCryptoType()
   const fiatType = normalizedQuote.getFiatType()
   if (flow === CICOFlow.CashOut) {
-    const receive = normalizedQuote.getFiatAmount()
-    const total = normalizedQuote.getCryptoAmount()
-    const fee = normalizedQuote.getFeeInCrypto(exchangeRates)
-    const totalMinusFee = Number(total) - Number(fee || 0)
-    const exchangeRate = Number(receive) / totalMinusFee
+    const providerFee = normalizedQuote.getFeeInCrypto(exchangeRates) || new BigNumber(0)
+    let networkFee = new BigNumber(0)
+    if (feeEstimate?.usdFee) {
+      networkFee =
+        convertLocalAmountToCurrency(
+          convertDollarsToLocalAmount(new BigNumber(feeEstimate.usdFee), exchangeRates.cUSD),
+          exchangeRates[cryptoType]
+        ) || networkFee
+    }
+
+    const receive = Number(normalizedQuote.getFiatAmount())
+    const total = Number(
+      new BigNumber(normalizedQuote.getCryptoAmount()).plus(networkFee).toString()
+    )
+    const totalFee = Number(providerFee.plus(networkFee))
+    const totalMinusProviderFee = total - totalFee
```

Comment:
nit: I think this should be named `totalMinusFees` because it includes both the provider and network fees

Reply:
why was this marked resolved? looks like the name is still totalMinusProviderFee

Reply:
:facepalm: Thanks for the catch... I made the changes and then stashed them to debug something but never re-applied it...